infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```diff
@@ -235,6 +235,30 @@ param parTags object = {}
 @sys.description('Set Parameter to true to Opt-out of deployment telemetry.')
 param parTelemetryOptOut bool = false
 
+@sys.description('Define inbound destination port or a range for HTTPS. Default:443')
+param parHttpsPort string = '443'
+
+@sys.description('Define inbound destination port or a range for gateway manager. Default:443')
+param parGatewayManagerPort string  ='443'
```

Comment:
does this really need to be a parameter. Bastion gateway manager is only accessible via 443 so therefore no customization option required.

Propose we remove and hardcode as before